controllers/argocd/route.go

```diff
@@ -325,8 +328,14 @@ func isCreatedByServiceCA(crName string, secret corev1.Secret) bool {
 
 // reconcileApplicationSetControllerWebhookRoute will ensure that the ArgoCD Server Route is present.
 func (r *ReconcileArgoCD) reconcileApplicationSetControllerWebhookRoute(cr *argoproj.ArgoCD) error {
-	name := fmt.Sprintf("%s-%s", common.ApplicationSetServiceNameSuffix, "webhook")
-	route := newRouteWithSuffix(name, cr)
+	// Generate a base name for the route
+	baseName := fmt.Sprintf("%s-%s", cr.Name, "applicationset-controller-webhook")
```

Comment:
Can we have `applicationset-controller-webhook` stored as a constant in `common` package. I think its used in multiple places. Also, can we make it shorter ? `appset-webhook` or `appset-wbhk`